jetty9-compat-base/src/main/java/com/google/apphosting/runtime/jetty9/SessionManager.java

```diff
@@ -148,8 +148,7 @@ public boolean isDirty() {
 
     @Override
     public void renewId(HttpServletRequest request) {
-
-      String oldId = getClusterId();
+      final String oldId = getClusterId();
```

Comment:
The final was suggested by the checkstyle warning because the assignment is too far from the usage.